email_mailbox/src/components/SettingAccountWrapper.js

```diff
@@ -723,6 +751,46 @@ class SettingAccountWrapper extends Component {
     );
   };
 
+  handleChangeSwitchBlockingRemote = ev => {
+    const nextValue = ev.target.checked;
+    this.setState(
+      {
+        blockRemoteContentParams: {
+          ...this.state.blockRemoteContentParams,
+          isLoading: true
+        }
+      },
+      async () => {
+        const { status } = await setBlockRemoteContent(nextValue);
+        const blockRemoteContentParams = {
+          ...this.state.blockRemoteContentParams
+        };
+        if (status === 200) {
+          let newState;
+          blockRemoteContentParams['blockRemoteContentEnabled'] = nextValue;
+          blockRemoteContentParams['isLoading'] = false;
+          await changeEmailBlockedAccount({
+            blockRemoteContent: nextValue
+          });
+          if (nextValue === true) {
+            sendBlockRemoteContentTurnedOn();
+          } else {
+            sendBlockRemoteContentTurnedOff();
+          }
+          // eslint-disable-next-line prefer-const
+          newState = {
```

Comment:
you don't need to declare `newState` above if you don't use it before this. Declare it as const